### DISCUSSION TITLE
Different output with keypad vs `C-c`

### DISCUSSION THREAD

**Discussion by gf3** [meow-edit/meow#242] [Q&A]
Is it possible to show the same output when using the keypad as when using `C-c`?

Currently <kbd>SPC</kbd> <kbd>c</kbd> just lists sub-commands as `+prefix` which is not very useful to me.

I'm using meow with Doom Emacs btw.

### Example

<kbd>SPC</kbd> <kbd>c</kbd>

![Screenshot 1](https://screenshot.pics/s/doom__Doom_Emacs____221__77_2022-05-04_16-49-21.png)

<kbd>C-c</kbd>

![Screenshot 2](https://screenshot.pics/s/doom__Doom_Emacs____221__77_2022-05-04_16-49-59.png)

**Comment by DogLooksGood** [comment]
How about `(setq meow-keypad-leader-dispatch doom-leader-map)`.

**Reply by gf3** [reply]
unfortunately it's the same:

![Screenshot](https://screenshot.pics/s/pagination_test.go__Doom_Emacs_2022-05-09_12-57-27.png)
